README.md

```diff
@@ -31,13 +31,13 @@ You'll notice that we used an HTML-like syntax; [we call it JSX](https://faceboo
 
 ## Installation
 
-The fastest way to get started is to serve JavaScript from the CDN (also available on [cdnjs](https://cdnjs.com/libraries/react) and [jsdelivr](https://www.jsdelivr.com/projects/react)):
+The fastest way to get started is to serve JavaScript from a CDN. We're using [npmcdn](https://npmcdn.com/) here but React is also available on [cdnjs](https://cdnjs.com/libraries/react) and [jsdelivr](https://www.jsdelivr.com/projects/react):
```

Comment:
"We're using [npmcdn](https://npmcdn.com/) here" can lead people to believe we use it at Facebook.
Can we reword to something like "We're using [npmcdn](https://npmcdn.com/) below"?